fetch.bs

```diff
@@ -5363,14 +5363,41 @@ returning a response directly, or allowing the request to proceed by returning n
      <li><p>Let <var>isTopLevelNavigation</var> be true if <var>request</var>'s
      <a for=request>destination</a> is "<code>document</code>"; otherwise, false.
 
-     <li><p>If <var>isTopLevelNavigation</var> is false and <var>requestEnvironment</var> is null,
-     then return a <a>network error</a>.
+     <li><p>Let <var>isTopLevelSelfFetch</var> be false.
+
+     <li><p>If <var>request</var>'s <a for=request>client</a> is non-null:
+      <ol>
+       <li><p>Let <var>global</var> be <var>request</var>'s <a for=request>client</a>'s
+       <a for="environment settings object">global object</a>.
+
+       <li>
+        <p>If all of the following conditions are true:
+
+         <ul class=brief>
```

Comment:
The `<ul>` should not be further indented. It becomes a sibling of the `<p>` after all.

Reply:
Done